ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -281,12 +285,11 @@ nnfw_tensor_info_copy (const nnfw_tensorinfo * nnfw_info,
   if ((status = nnfw_tensor_type_to_gst (nnfw_info->dtype, &gst_info->type)) < 0)
     return status;
 
-  /** reverse the order of dimension */
-  for (idx = 0; idx < nnfw_info->rank; idx ++)
-    gst_info->dimension[NNS_TENSOR_RANK_LIMIT - idx - 1] = nnfw_info->dims[idx];
+  for (idx = nnfw_info->rank-1; idx >= 0; idx--)
```

Comment:
check the style (gst-indent)

Reply:
Can you sure that idx won't go beyond 3?

Reply:
> check the style (gst-indent)

Updated

Reply:
> Can you sure that idx won't go beyond 3?

Yes, line 281 does this check on the rank of nnfw_info.